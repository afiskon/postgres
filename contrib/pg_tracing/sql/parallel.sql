begin;
-- encourage use of parallel plans
set parallel_setup_cost=0;
set parallel_tuple_cost=0;
set min_parallel_table_scan_size=0;
set max_parallel_workers_per_gather=4;
/*dddbs='postgres.db',traceparent='00-00000000000000000000000000000001-0000000000000001-01'*/ select 1 from pg_class limit 1;
/*dddbs='postgres.db',traceparent='00-00000000000000000000000000000002-0000000000000002-01'*/ select 2 from pg_class limit 2;
/*dddbs='postgres.db',traceparent='00-00000000000000000000000000000003-0000000000000003-00'*/ select 3 from pg_class limit 3;
commit;

SELECT resource from pg_tracing_spans order by resource;
