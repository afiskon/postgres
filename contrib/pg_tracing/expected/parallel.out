begin;
-- encourage use of parallel plans
set parallel_setup_cost=0;
set parallel_tuple_cost=0;
set min_parallel_table_scan_size=0;
set max_parallel_workers_per_gather=4;
/*dddbs='postgres.db',traceparent='00-00000000000000000000000000000001-0000000000000001-01'*/ select 1 from pg_class limit 1;
 ?column? 
----------
        1
(1 row)

/*dddbs='postgres.db',traceparent='00-00000000000000000000000000000002-0000000000000002-01'*/ select 2 from pg_class limit 2;
 ?column? 
----------
        2
        2
(2 rows)

/*dddbs='postgres.db',traceparent='00-00000000000000000000000000000003-0000000000000003-00'*/ select 3 from pg_class limit 3;
 ?column? 
----------
        3
        3
        3
(3 rows)

commit;
SELECT resource from pg_tracing_spans order by resource;
             resource              
-----------------------------------
 End
 End
 End
 End
 End
 End
 End
 End
 Finish
 Finish
 Finish
 Finish
 Finish
 Finish
 Finish
 Finish
 Gather
 Gather
 Limit
 Limit
 Parallel SeqScan on pg_class
 Parallel SeqScan on pg_class
 Parallel SeqScan on pg_class
 Parallel SeqScan on pg_class
 Parallel SeqScan on pg_class
 Parallel SeqScan on pg_class
 Parallel SeqScan on pg_class
 Parallel SeqScan on pg_class
 Parse
 Parse
 Planner
 Planner
 Post Parse
 Post Parse
 Run
 Run
 Run
 Run
 Run
 Run
 Run
 Run
 Start
 Start
 Start
 Start
 Start
 Start
 Start
 Start
 Worker 0
 Worker 0
 Worker 1
 Worker 1
 Worker 2
 Worker 2
 select $1 from pg_class limit $2;
 select $1 from pg_class limit $2;
(58 rows)

